### DISCUSSION TITLE
run `down` block for specific migration with `Sequel::TimestampMigrator`

### DISCUSSION THREAD

**Discussion by opya** [jeremyevans/sequel#2045] [General]
Hi guys,

I was wondering how to run `down` block for specific migration with `Sequel::TimestampMigrator`. I'm trying like that: 

```ruby
Sequel::TimestampMigrator.run(DB, ' ./db/migrations', target: 1685427879, direction: :down, table: 'schema_migrations')

```

**Comment by jeremyevans** [comment]
This isn't currently supported.  You can only provide a target, and the migrator will apply the necessary changes to reach that target version based on the contents of the migration directory.  Maybe we could add support for:

```ruby
Sequel::TimestampMigrator.apply(DB, 'db/migrations/1234_migration.rb', direction: :down, table: 'schema_migrations')
```

**Reply by jeremyevans** [reply]
Well, `apply` is already used, so maybe `run_single`.
